app/pathogen/sarscov2/dashboard/(map)/MapAndFilters.tsx

```diff
@@ -95,9 +95,13 @@ export default function MapAndFilters() {
                   },
                 },
               ]}
-              generatePopupContent={(record) => (
-                <SarsCov2StudyPopupContent record={record} />
-              )}
+              generatePopupContent={(input) => {
+                if(input.layerId === 'country-highlight-layer') {
+                  throw new Error("The SARS CoV2 Country pop-up is unimplemented");
+                }
```

Comment:
Attaining maximum laziness level lmao

In all seriousness, we'll come back for this when we move the CoV2 stuff over hahaha
It's feature flagged off so we're 100% cool. I'm just doing this because the types are `any` and it isn't clear you have to do this if I don't explicitly write it like this.